src/pages/index.tsx

```diff
@@ -1,16 +1,76 @@
-import { Inter } from "next/font/google";
+import Image from "next/image";
 
-import Button from "@/components/Button";
-
-const inter = Inter({ subsets: ["latin"] });
+import Button from "@/components/Button/index";
 
 export default function Home() {
   return (
-    <main
-      className={`flex min-h-screen flex-col items-center justify-between p-24 ${inter.className}`}
-    >
-      <h1>Hello!</h1>
-      <Button aria-label="button"> Hello </Button>
-    </main>
+    <div className="bg-lightAqua-100">
+      <div className="relative h-44 w-full">
+        <Image
+          src="/images/landing.jpg"
+          alt="Greyscale picture of water animal"
+          fill={true}
+          className="object-cover object-center"
+        />
+      </div>
+      <div className="relative z-10 -mt-16 flex w-full justify-center">
+        <Image
+          src="/images/repair_lab_logo.png"
+          alt="Greyscale picture of water animal"
+          width={500}
+          height={500}
+          className="h-28 w-28"
```

Comment:
In your Image components, you've added classes like `h-28 w-28` but have defined the width and height. Next.js's Image component doesn't make use of these classes since it has its own width and height properties. 

However, deleting the className will make the logo be 500px and I assume you want it to be `h-28 w-28` which corresponds to 112px.

```
width={112}
height={112}
```